var/spack/repos/builtin/packages/py-netcdf/package.py

```diff
@@ -0,0 +1,38 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyNetcdf(PythonPackage):
+    """Python interface to the netCDF Library."""
+    homepage = "http://unidata.github.io/netcdf4-python"
+    url      = "https://github.com/Unidata/netcdf4-python/tarball/v1.2.3.1rel"
+
+    version('1.2.3.1', '4fc4320d4f2a77b894ebf8da1c9895af')
+
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-cython', type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('netcdf')
```

Comment:
I'm seeing a lot of other dependencies here:
```python
HDF5_dir = os.environ.get('HDF5_DIR')                                           
HDF5_incdir = os.environ.get('HDF5_INCDIR')                                     
HDF5_libdir = os.environ.get('HDF5_LIBDIR')                                     
netCDF4_dir = os.environ.get('NETCDF4_DIR')                                     
netCDF4_incdir = os.environ.get('NETCDF4_INCDIR')                               
netCDF4_libdir = os.environ.get('NETCDF4_LIBDIR')                               
szip_dir = os.environ.get('SZIP_DIR')                                           
szip_libdir = os.environ.get('SZIP_LIBDIR')                                     
szip_incdir = os.environ.get('SZIP_INCDIR')                                     
hdf4_dir = os.environ.get('HDF4_DIR')                                           
hdf4_libdir = os.environ.get('HDF4_LIBDIR')                                     
hdf4_incdir = os.environ.get('HDF4_INCDIR')                                     
jpeg_dir = os.environ.get('JPEG_DIR')                                           
jpeg_libdir = os.environ.get('JPEG_LIBDIR')                                     
jpeg_incdir = os.environ.get('JPEG_INCDIR')                                     
curl_dir = os.environ.get('CURL_DIR')                                           
curl_libdir = os.environ.get('CURL_LIBDIR')                                     
curl_incdir = os.environ.get('CURL_INCDIR')    
```
Some of these even specify which version is required:
```python
'HDF5 version >= 1.8.0 is required'
```